### PR TITLE
Add logos to top right, Add initial load animation, Current node name in toolbar 

### PR DESCRIPTION
**Pull request by gotwig**

### CODE REVIEW

File: index-debug.html [hunk 1]
```diff
@@ -82,7 +82,14 @@
...
+
+        w2utils.lock(loadingDiv, myHTML, true);
```
Comment (julien-moreau) [inline index-debug.html]: Do we see the spinner with that logo ? It is set to « true » so you have the spinner on the left and the logo on the right ?

File: index-debug.html [hunk 2]
```diff
@@ -82,7 +82,14 @@
 
         // Lock and import
         var loadingDiv = document.getElementById('LOADING-DIV');
-        w2utils.lock(loadingDiv, 'Loading Editor...', true);
+
+        var myHTML = `<style>
```
Comment (julien-moreau) [inline index-debug.html]: Please use a correct variable name :)

File: css/editor.css [hunk 3]
```diff
@@ -1,5 +1,18 @@
...
...fq8Ei+trhSXqBewURo/Q2ZmPutRMB5Y/XLp6fQcv/MfzOO6m8u/MoXZ3y73t2ODCGuuaPVJJIWnRO5yj+NGf7kgNbhKWWJAkEU+eozPM+80Qjh+UVchVW9eHLUXN++cEZRqLdB/JrQDkoxpF4gTOaGJ7Iru1/3Vw18cBgV3MV4Wl/+1Y6c+TEYm3yzsnOsqqzp6ezCVRUsuSEQPVmKOdy4yO2dVJXal7FEgiAgeFn+/ol+bkJcrcgOnZ3z973fBXd3sdhRFZ9J0roiWY72FwyB3Sd6a/Z/NROP68s/HpVB9Tjr2/XeDte2YWVJ3cYlLGUgNA5v0bV40PwFGpH7nDgQhGVd14rQ0McTa46cE6+uXanoHHtU63lIYEDlCLGShi2CZV3n3d7e8ZbqvZ9ORUZnLsc1+8PH2+Zg7NQ7qP7W2B/vz1aVBQWWUhCqEhtC/DXDoudk8QKc+w5GEIkKZnE5yPPNLmhVD07/gKCz297af6s7esAXjE0drh3++uIDh1A9xlf98dnfeRgoPa4n92ks67uWqm0BlnoQ98FN55szVw8gYKuDH1yQawxUDRKJC9+1dO8YcWLG971YvGogfGhdz0P85uIRP14xonL8KTR4aqKya6we0Zds84NtAksZCKXWmKVUX7snmG+yILvorzIiiyp+r1iF9T5GEIkOVu0EbGq0YEYoFr24PvzGbxDR72CgDNk3D5hztNoUdX4Jjc27Fi3w81jJbNG5g60IZL8t3qHG5ZoLCB0bWbJAEPc/Fl0S6Ds+wE2M+GZ/M1dCez750NPzVqOlMXqXepVLwlIaQueqbr/ea4gI5ofn33JOIgjC2tJ7G1bgTt5I+wvn+IK/7/ige/tIsSHcdDNLb0gIkRvElZzP1HavLc9gEFgyQhCrt+yVwzw5d93qb3R2Du7xR96dxzbYW/ukOAaQhu0vgR1pI6727MYc8JjBGzmK/egeGCWBAnOSzwQJosAalpRHRysiB6Yv/10AcbHmEmZ/g3CULdb13bforT4JS2sImUwmyVXkLZbdq1gkl8tZykAQpkhzhvfZ8Ta+eQLz41dkDU/jfuGG1R177imu27SIpQMEQRAla9tuwu5wm3vrfo+leeeyAksg/ao/giAIna+OaZwUfP6jPTgWAAAAABjkbz2MPdUMAAAAAAAAAAAAAtrcybaWVsZtAAAAAElFTkSuQmCC);
```
Comment (julien-moreau) [inline css/editor.css:23]: Why repeating background image in editor.css ?

File: index-debug.html [hunk 4]
```diff
@@ -82,7 +82,13 @@
 
         // Lock and import
         var loadingDiv = document.getElementById('LOADING-DIV');
-        w2utils.lock(loadingDiv, 'Loading Editor...', true);
+
+        var initialLoadHTML = `<style>
```
Comment (julien-moreau) [inline index-debug.html]: I think we should override w2ui spinner to have a babylonjs logo instead :)

File: css/editor.css (hunk 3 above)
Comment (gotwig) [reply css/editor.css:23]: Its the logo for the top right. What you mean with repeating? You need a logo for white and black theme

File: index-debug.html (hunk 4 above)
Comment (gotwig) [reply index-debug.html]: Ok I did this :) will commit soon

File: css/editor.css (hunk 3 above)
Comment (julien-moreau) [reply css/editor.css:23]: I mean you also have the Babylonjs logo base64String string also in dark.css in this pull request

File: index-debug.html (hunk 4 above)
Comment (julien-moreau) [reply index-debug.html]: So then we’ll always have the logo has spinner

File: src/tools/behavior/code.ts [hunk 5]
```diff
@@ -233,14 +233,17 @@ export default class BehaviorCodeEditor extends EditorPlugin {
...
         // Select first behavior
         if (this.datas.metadatas.length > 0) {
             this.selectCode(0);
             this.grid.select([0]);
+            currentNodeName = this.datas.metadatas[0].name;
         }
 
         // Refresh right text
-        this.toolbar.element.right = `Attached to "${(node instanceof Scene ? 'Scene' : node.name)}"`;
+        this.toolbar.element.right = `<h2 id="currentNodeNameCode">` + currentNodeName + `</h2> Attached to "${node instanceof Scene ? 'Scene' : node.name}"`;
```
Comment (julien-moreau) [inline src/tools/behavior/code.ts:246]: I think this should moved to ".selectCode" so you'll not have getElementById :)

**Pull request merged**